### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sebhg**
just to test pull requests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # dummy
-for basic dummy tests with GitHub
+for basic dummy tests with GitHub and branches
```
Comment (sebhg) [inline README.md]: what else should I test ?

Comment (sebhmg) [inline README.md]: whatever

Comment (sebhg) [inline README.md]: hu?

Comment (sebhg) [inline README.md]: yep

**Pull request merged**